javascript/code-challenges/challenges-02.test.js

```diff
@@ -0,0 +1,321 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+
+CHALLENGE 1 - Review
+
+Write a function named raisedToTheThird that takes in an array of numbers and returns a new array of each of those numbers raised to the 3rd power (hint: look up Math.pow()). Use forEach to solve this problem.
+
+------------------------------------------------------------------------------------------------ */
+
+const raisedToTheThird = (arr) => {
+  // Solution code here...
```

Comment:
Hey there!

Really nice work on these 👍  This may seem silly, but go ahead and start removing these lines of commented out code in the challenges. No need to keep these hangin' around in all of your lovely code.